starter-1/pkg/cmd/root/root.go

```diff
@@ -0,0 +1,26 @@
+package root
+
+import (
+	"github.com/aerogear/charmil/core/factory"
+	"github.com/spf13/cobra"
+
+	Cliversion "github.com/aerogear/charmil/starter-1/pkg/cmd/version"
```

Comment:
`Nitpick`: Maybe we can convert this alias to lowercase. Almost every project that I've come across follows this naming convention. 
```suggestion
	cliversion "github.com/aerogear/charmil/starter-1/pkg/cmd/version"
```


WDYT?

Reply:
sure